src/main/java/stsjorbsmod/powers/MirroredTechniquePower.java

```diff
@@ -0,0 +1,79 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.util.CardMetaUtils;
+import stsjorbsmod.util.IntentUtils;
+import stsjorbsmod.util.ReflectionUtils;
+
+public class MirroredTechniquePower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(MirroredTechniquePower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    private static int extraPlays;
```

Comment:
I think it would be preferable for this to be non-static; this feels like it should be part of a given instance of the power to me, not shared among all instances. To achieve the desired stacking behavior, you'd want to pair "making this nonstatic" with moving that `Math.max` call out of the ctor and into an override of the `stackPower` method

Reply:
Makes sense, thanks.